### PR TITLE
Show human-readable timestamp for long-running commands

### PR DESCRIPTION
**Pull request by jedahan**
screen shot 2016-10-16 at 1 43 56 pm
This lets you see long running commands in rprompt, like in pure.

### CODE REVIEW

**Comment by frm** [comment]
Hi again! Glad to see you around!

I really like this but I'm wondering if we should add a separator between the command duration and the branch. What do you think?

File: geometry.zsh [hunk 1]
```diff
@@ -49,10 +49,35 @@ PROMPT_GEOMETRY_GIT_CONFLICTS=${PROMPT_GEOMETRY_GIT_CONFLICTS:-false}
 PROMPT_GEOMETRY_GIT_TIME=${PROMPT_GEOMETRY_GIT_TIME:-true}
 PROMPT_GEOMETRY_COLORIZE_SYMBOL=${PROMPT_GEOMETRY_COLORIZE_SYMBOL:-false}
 PROMPT_GEOMETRY_COLORIZE_ROOT=${PROMPT_GEOMETRY_COLORIZE_ROOT:-false}
+PROMPT_GEOMETRY_COMMAND_MAX_EXEC_TIME=${PROMPT_GEOMETRY_COMMAND_MAX_EXEC_TIME:-5}
 
 # Use ag if possible
 GREP=$(command -v ag >/dev/null 2>&1 && echo "ag" || echo "grep")
 
+# from https://github.com/sindresorhus/pretty-time-zsh
+prompt_geometry_seconds_to_human_time() {
```
Comment (frm) [inline geometry.zsh:59]: One last thing before merging. Since you added this function, can you update the git commit time function to use it as well?

**Comment by jedahan** [comment]
welp i screwed up this pull request, creating another...

**Comment by frm** [comment]
Closing for #22

**Pull request closed without merging**